scripts/linux/runE2ETest.sh

```diff
@@ -519,6 +519,7 @@ function run_longhaul_test() {
         -t "$ARTIFACT_IMAGE_BUILD_NUMBER-linux-$image_architecture_label" \
         --leave-running=All \
         -l "$deployment_working_file" \
+        --runtime-log-level "Info" \
```

Comment:
This should be default. Do we still need to set it?

Reply:
No; Debug is the default, which is good for e2e tests.  https://github.com/Azure/iotedge/blob/0c00a2893f89c547201d34fe8011195ae24fa9ab/smoke/IotEdgeQuickstart/Program.cs#L112